cpp/src/parquet/column_reader.cc

```diff
@@ -23,13 +23,15 @@
 #include <exception>
 #include <iostream>
 #include <memory>
+#include <optional>
```

Comment:
So this is unrelated? ( but actually it's required by this file?)

Reply:
Yes, I'll remove, sorry.